docs/sql/mysql/create_schema.sql

```diff
@@ -204,6 +204,42 @@ CREATE TABLE `pa_recovery_config` (
   CONSTRAINT `FK_RECOVERY_CONFIG_APP` FOREIGN KEY (`application_id`) REFERENCES `pa_application` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
 ) ENGINE=InnoDB AUTO_INCREMENT=1 CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
+--
+-- Create table for operations
+--
+CREATE TABLE pa_operation (
+    id varchar(37) NOT NULL,
+    user_id varchar(255) NOT NULL,
+    application_id bigint(20) NOT NULL,
+    template_id bigint(20) NULL,
+    external_id varchar(255) NULL,
+    operation_type varchar(255) NOT NULL,
+    data text NOT NULL,
+    parameters text NULL,
+    status int(11) NOT NULL,
+    signature_type varchar(255) NOT NULL,
+    failure_count bigint(20) default 0 NOT NULL,
+    max_failure_count bigint(20) NOT NULL,
+    timestamp_created datetime NOT NULL,
+    timestamp_expires datetime NOT NULL,
+    timestamp_finalized datetime NULL,
+    PRIMARY KEY (id)
+) ENGINE=InnoDB AUTO_INCREMENT=1 CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+--
+-- Create table for operation templates
+--
+CREATE TABLE pa_operation_template (
+    id bigint(20) NOT NULL,
+    template_name varchar(255) NOT NULL,
+    operation_type varchar(255) NOT NULL,
```

Comment:
The `operation_type` field name is a bit unfortunate, because it creates an inconsistency between Next Step operations and PowerAuth operations:
- In Next Step, we use `operation_type` in steps, the value is either `CREATE` or `UPDATE`, see: https://github.com/wultra/powerauth-webflow/blob/fc2b4b8aeab2fb9dcb46c48b7d0e1867ce45f261/docs/sql/mysql/create_schema.sql#L169
- The `type` suggests that there are several operation types available, hence an enumeration can be used. However, the value cannot be typized because we do not know the value upfront.
- We use `operation_name` field for the same purpose in the rest of PowerAuth stack. While this field name is also not perfect, I think it is better to keep the terminology for consistency reasons.

I would like to ask, if it is still possible to rename this field? It will be a bit hard to explain the relationship between `operation_name` and `operation_type` in the documentation. I understand that we may have already deployed this DB structure, but I think a migration is possible before the final release and we can avoid headaches in the future. 

Thank you for considering this difficult change.